src/js/popup/view/view.js

```diff
@@ -1,82 +0,0 @@
-/**
- * @file Contains UI of popup
- */
-
-import { isTimerActive } from "../actions/timer";
-import { updateTimerCounter } from "./timerCounterView";
-
-import imgStart from "../../../img/start-gray-16.png";
-import imgStop from "../../../img/stop-red-16.png";
-
-const timerBtn = document.getElementById("timerToggler");
-
-function updateView(data) {
-  updateButtonView(isTimerActive(data));
-  updateInputsView(data.timer.projectId, data.timer.mergeRequestId);
-  updateTimerCounter(data);
-}
-
-function updateInputsView(projectId, mergeRequestId) {
-  if (projectId == undefined || mergeRequestId == undefined) {
-    return;
-  }
-  setProjectId(projectId);
-  setMergeRequestId(mergeRequestId);
-}
-
-function updateButtonView(started) {
-  const timerButtonIcon = document.getElementsByClassName(
-    "timer-toggler-icon"
-  )[0];
-
-  if (started) {
-    timerButtonIcon.classList.replace("fa-play-circle", "fa-stop-circle");
-    browser.browserAction.setIcon({ path: imgStop });
```

Comment:
Please reapply these lines.

After the change icon is not changing when timer is started and tooltip is not changed either

Reply:
Good catch!